receiver.go

```diff
@@ -216,24 +214,14 @@ func (r *Receiver) Listen(ctx context.Context, handler Handler) *ListenerHandle
 	ctx, span := r.startConsumerSpanFromContext(ctx, "sb.Receiver.Listen")
 	defer span.End()
 
-	messages := make(chan *amqp.Message)
```

Comment:
no more channel to manage. we don't need an implicit buffer. prefetch messages are already buffered in `go-amqp`